### PR TITLE
add mousetrap's git auto-update & fix v1.4.6 structure 

### LINKED ISSUE #9896: [Request] Add Mousetrap latest version
**Library name: Mousetrap
**Git repository url: https://github.com/ccampbell/mousetrap

Project is up to 1.6.0 but the latest available here is 1.4.6

### PR DESCRIPTION
**Pull request by pvnr0082t**
Pull request for issue: #9896 
@kennynaoh @x09326 Please help me review this pull request, thank you.
- Using git auto-update due to there are 23 releases on GitHub but 6 onNPM
- Manually add v0.0.1 from NPM due to git auto-update can't get.
- Make mousetrap v1.4.6 file structure as the same as the latest version

close #9896

### CODE REVIEW

**Comment by kennynaoh** [comment]
LGTM

File: ajax/libs/mousetrap/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "mousetrap",
   "filename": "mousetrap.min.js",
-  "version": "1.4.6",
+  "version": "1.6.0",
   "description": "Simple library for handling keyboard shortcuts",
   "homepage": "http://craig.is/killing/mice",
```
Comment (x09326) [inline ajax/libs/mousetrap/package.json]: @pvnr0082t 
Let's use `https` instead of `http`.

Comment (pvnr0082t) [inline ajax/libs/mousetrap/package.json]: @x09326 Same as #9895. It will redirect to `https://craig.is/killing/mice`. I think we don't need to modify it.

Comment (x09326) [inline ajax/libs/mousetrap/package.json]: Okay

**Comment by x09326** [comment]
@pvnr0082t 
Please rebase with the latest master branch, thanks.

**Comment by pvnr0082t** [comment]
@kennynaoh @x09326 I have rebased it. Please help me review again, thank you.

**Comment by x09326** [comment]
LGTM
ping @PeterDaveHello

**Comment by pvnr0082t** [comment]
@PeterDaveHello I changed the structure for v1.4.6 as the same as new version structure. What do you mean "broken"?

**Comment by PeterDaveHello** [comment]
@pvnr0082t What about the users using the old url? You'll break their website.

**Comment by PeterDaveHello** [comment]
@pvnr0082t You can only add the correct one, can't break the old one.

**Comment by pvnr0082t** [comment]
@x09326 @kennynaoh I have add new structure for v1.4.6 without making it broken. Please help me review the PR again, thank you.

**Comment by PeterDaveHello** [comment]
Nice!

**Pull request merged**